samples/bluetooth/peripheral_hids_keyboard/prj.conf

```diff
@@ -43,3 +43,6 @@ CONFIG_SETTINGS=y
 CONFIG_SETTINGS_FCB=y
 
 CONFIG_DK_LIBRARY=y
+
+# When using BT_LL_NRFXLIB, the slave count must be >=2 in this example.
+CONFIG_BLECTRL_SLAVE_COUNT=2
```

Comment:
Does this setting apply? Nordic LL is not the default Controller used in the BLE samples, so by default `CONFIG_BLECTRL_SLAVE_COUNT` is not defined, so cannot be overridden and will produce a Kconfig warning.
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1406 would solve the config problem in this example.

Reply:
If https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1406 comes to a sensible conclusion it may resolve the issue addressed here (except for the workqueue stack). https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1406/commits/0bd9d62996136785d454c082f5e50f3ec7afff04 actually resolves this because it sets `CONFIG_BLECTRL_SLAVE_COUNT` to `CONFIG_BT_MAX_CONN` in this sample but it is a bit tricky and we'll see if it gets accepted.

Reply:
Maybe we should add prj_[board].conf configuration files?
We should not add anything to configuration files that would generate warnings when project is compiled out of the box, without any changes from the user. It is just ugly and make user disturbed.

Reply:
@rakons, you mean `prj_bt_ll_nrfxlib.conf` vs `prj_bt_ll_sw.conf`? Board has no impact here. Splitting out the configurations will also cause a sync issue. I agree that we don't want any warnings when compiling out of the box, so we'll have to change this.

Reply:
This change has been taken out of this PR and will be handled in #1406